### PR TITLE
incorporated findings from doing binding code reviews

### PR DESCRIPTION
**Pull request by kaikreuzer**
Signed-off-by: Kai Kreuzer <kai@openhab.org>

### CODE REVIEW

**Comment by kaikreuzer** [comment]
@kubawolanin FYI - this should cover most stuff that I commented on https://github.com/openhab/openhab2-addons/pull/1738#pullrequestreview-21451728

File: developers/contributing/contributing.md [hunk 1]
```diff
@@ -166,6 +160,12 @@ then you just add a line to every git commit message:
 
 using your real name (sorry, no pseudonyms or anonymous contributions.)
 
+If your commit contains code from others as well, please ensure that they certify the DCO as well and add them with an "Also-By" line to your commit message:
+
+    Also-by: Ted Nerd <ted.nerd@email.com> (github: github_handle_ted)
+    Also-by: Sue Walker <sue.walker@email.com> (github: github_handle_sue)
+    Signed-off-by: Joe Smith <joe.smith@email.com> (github: github_handle_joe)
```
Comment (kubawolanin) [inline developers/contributing/contributing.md]: Does the order matter here? IMO the following makes more sense:

```
Signed-off-by: Joe Smith <joe.smith@email.com> (github: github_handle_joe)
Also-by: Ted Nerd <ted.nerd@email.com> (github: github_handle_ted)
Also-by: Sue Walker <sue.walker@email.com> (github: github_handle_sue)
```

Comment (kaikreuzer) [inline developers/contributing/contributing.md]: Yes, at least that's also how the Eclipse Foundation handles it.
The Signed-off-by should always be the last line (like a signature on a document).

**Comment by ThomDietrich** [comment]
LGTM

**Pull request merged**